serverless/architecture/serverless-serving-architecture.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-serving-architecture"]
+= Knative Serving architecture
+:context: serverless-serving-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Serving supports deploying and managing serverless applications, by providing a set of objects as Kubernetes CustomResourceDefinitions (CRDs) that define and control the behavior of serverless workloads on an {product-title} cluster.
```

Comment:
This is specific to "Serverless" as a whole though, whereas this section is specifically just talking about Knative Serving. We explain Serverless on the main getting started page also. Gonna leave this as is for now.

Reply:
The main getting started page is below the Architecture. Also, the getting started doesn't say what is Serverless and it's main component - serving. There are no links to Red Hat topics on Cloud-Native apps and Serverless architecture.

That should be somewhere. This section is about architecture. That's an engineering detail, that should be on _"page 2"_. In my opinion it would be good to structure the docs in this way:

- "page 1" - about serverless project - content designed for everyone (decision makers, marketing & PR, architects, devs, ops)
- "page 2" - serverless architecture - content designed for technical conceptual personnel (architects, devs, ops)
- "page 3-n" - serverless implementation in detail - content designed for technical workers (devs, ops)

Reply:
The main getting started page is below the Architecture. Also, the getting started doesn't say what is Serverless and it's main component - serving. There are no links to Red Hat topics on Cloud-Native apps and Serverless architecture.

That should be somewhere. This section is about architecture. That's an engineering detail, that should be on _"page 2"_. In my opinion it would be good to structure the docs in this way:

- "page 1" - about serverless project - content designed for everyone (decision makers, marketing & PR, architects, devs, ops)
- "page 2" - serverless architecture - content designed for technical conceptual personnel (architects, devs, ops)
- "page 3-n" - serverless implementation in detail - content designed for technical workers (devs, ops)

Reply:
This PR is blocking me from rebasing another whole PR on top of it relating to updates for the getting started section #24027 just FYI.

Please keep QE reviews to the scope of the content going forward rather than comments about structure etc. A PR is not the place to raise this - if there are changes to structure etc for docs you'd like to see let's discuss it in some other forum.

Reply:
Sure, but I'm opting here to include the following, which isn't present anywhere. 

> Knative Serving on {product-title} enables developers to write a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native applications] using a link:https://www.redhat.com/en/topics/cloud-native-apps/what-is-serverless[serverless architecture].
> Serverless refers to a cloud computing model whereby application developers don't have to provision servers or manage scaling for their app. Instead, those routine tasks are abstracted away by the platform, allowing developers to push code to production much faster than in traditional models.

The Serving architecture section is lacking this overall view (what? why? questions), but it starts with saying how knative delivers serverless (the implementation).

Getting started doesn't contain that architectural view either. Also, I wouldn't look at getting started for architectural overview, but here: Architecture > Serving.

Reply:
OK this I do really like, thank you :slightly_smiling_face: added.